src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit';
```

Comment:
- [ ] Kindly import `combineReducers` and `createStore` from `'redux'`. Currently, you are importing configureStore from `'@reduxjs/toolkit'`. Remember that the essence of this milestone is to use `react-redux` and not `redux/toolkit`. Kindly implement the change. 👍

Reply:
@Gambit142 Thank you very much for your feedback. I will change this now.